images/devtools-terraform-v1beta1/context/trivy-tf-scan

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# a flag to track errors
+error_occurred=false
+
+scan_file() {
+  local file="$1"
+  echo "Scanning $file"
+
+  if ! trivy --cache-dir /tmp/trivy/ config --exit-code 1 "$file"; then
+    error_occurred=true
+  fi
+}
+
+scan_directory() {
+  local dir="$1"
+  for tf_file in $(find "$dir" -type f \( -name "*.tf" -o -name "*.lock.hcl" \)); do
```

Comment:
```suggestion
  for tf_file in "$(find "$dir" -type f \( -name "*.tf" -o -name "*.lock.hcl" \))"; do
```
You might want to wrap the thing in quotes, just in case